unittests/fixtures/factory/work-dir/update_proxy.py

```diff
@@ -6,16 +6,16 @@
 """This file is specialized in updating a proxy file
 
 
- All information is passed via the environment;
-  so it has no arguments
+All information is passed via the environment;
+ so it has no arguments
```

Comment:
Is this intended to be the structure for this part of the documentation? Is it following the max chars per line limit here which is why this is splitting into two lines?